### PR TITLE
Dropdown on navigation header

### PR DESCRIPTION
**Pull request by AntonioLok**
# Changelog
- Added a dropdown when a user hover on one of the sections

# Test
- Install dependencies using `npm i`
- start app using `npm run start-webapp`
- navigate to `http://localhost:8080/`
- hover on one of the sections, a drop down will appear with the different type of clothing. Clicking on one of it will take you to the product page.

### CODE REVIEW

File: src/components/common/nav-header/index.js [hunk 1]
```diff
@@ -1,17 +1,78 @@
...
+
+    return (
+      <ButtonToolbar>
+        <DropdownButton
+          bsStyle="nav-header"
+          title={section}
+          noCaret
+          onMouseEnter={() => this.enterMenu()}
+          onMouseLeave={() => this.leaveMenu()}
+          open={show}
```
[the remark this answers is not in the document]
Comment (AntonioLok) [reply src/components/common/nav-header/index.js]: Yup, that's correct.

[the remark this answers is not in the document]
Comment (howardhenry) [reply src/components/common/nav-header/index.js]: Ok. The solution here seems a bit hackish as this react bootstrap component was meant to be exposed on click. But ok

File: src/components/common/nav-header/index.js [hunk 2]
```diff
@@ -1,17 +1,78 @@
...
+    history.push(`/${section}/${eventKey}`);
+  }
+
+  render() {
+    const { section } = this.props;
+    const { show } = this.state;
+
+    return (
+      <ButtonToolbar>
+        <DropdownButton
```
[the remark this answers is not in the document]
Comment (AntonioLok) [reply src/components/common/nav-header/index.js]: This was done in one of the examples that they provided https://react-bootstrap.github.io/components/dropdowns/

[the remark this answers is not in the document]
Comment (howardhenry) [reply src/components/common/nav-header/index.js]: As yes you're right 👍

File: src/components/common/nav-header/index.js [hunk 3]
```diff
@@ -1,17 +1,78 @@
...
+          id={section}
+        >
+          {TYPES[section].map(type => renderSections(type))}
+        </DropdownButton>
+      </ButtonToolbar>
+    );
+  }
+}
+
+const NavHeader = (props) => {
```
[the remark this answers is not in the document]
Comment (AntonioLok) [reply src/components/common/nav-header/index.js]: Okay. I just pushed my changes to address this issue. Please let me know if what I committed is good so I can go ahead and merge it with the master.

**Pull request merged**